### PR TITLE
[GP-3106] bigint ids

### PR DESCRIPTION
**Pull request by avarsava**
jooq maps these to Long. postgres already set up the sequence behind the auto-increment as bigint for some reason so only the surface level types need to be changed.

### CODE REVIEW

File: vidarr-server/src/main/resources/db/migration/V0010__big_serial_id.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE workflow_run ALTER COLUMN id TYPE bigint;
+ALTER TABLE active_workflow_run ALTER COLUMN id TYPE bigint;
+ALTER TABLE active_operation ALTER COLUMN workflow_run_id TYPE bigint;
```
Comment (callunity) [inline vidarr-server/src/main/resources/db/migration/V0010__big_serial_id.sql:3]: @mlaszloffy can you confirm if active_operation.id should also be switched over to bigint? It's currently at least 212,662,776

Comment (mlaszloffy) [reply vidarr-server/src/main/resources/db/migration/V0010__big_serial_id.sql:3]: Good catch, I am seeing the last sequence value at 2,079,044 - but this id will grow quickly too. Makes sense to change this ID too.

Comment (avarsava) [reply vidarr-server/src/main/resources/db/migration/V0010__big_serial_id.sql:3]: Will do

**Comment by avarsava** [comment]
Docker is broken on my laptop to the point where i can't fully build vidarr because the tests crash so i need to push to github just to pull on desktop. I know the checks are failing it's fine I'll fix it :+1:

**Review by mlaszloffy — APPROVED** [review]

**Review by callunity — APPROVED** [review]

**Pull request merged**